core/java/iesi-rest-without-microservices/.gitignore

```diff
@@ -12,6 +12,7 @@ HELP.md
 /classes/
 !.mvn/wrapper/maven-wrapper.jar
 iesi-home.conf
+test.mv.db
 
```

Comment:
Has the file "test.mv.db" been committed on purpose before adding it to the gitignore ?

Reply:
no, I'll purge it from the Git repository